src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java

```diff
@@ -301,6 +315,19 @@ private DlpServiceSettings getSettings() throws IOException {
       return transformationConfigs;
     }
 
+    public HashSet<String> getRequiredFields() {
+
+      HashSet requiredFields = new HashSet<String>();
```

Comment:
Missing type parameter in the declaration.

Reply:
added